[debug] [2019-07-31T11:42:06.027Z] ----------------------------------------------------------------------
[debug] [2019-07-31T11:42:06.030Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\14pra\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login
[debug] [2019-07-31T11:42:06.030Z] CLI Version:   7.2.2
[debug] [2019-07-31T11:42:06.030Z] Platform:      win32
[debug] [2019-07-31T11:42:06.031Z] Node Version:  v10.16.0
[debug] [2019-07-31T11:42:06.032Z] Time:          Wed Jul 31 2019 17:12:06 GMT+0530 (India Standard Time)
[debug] [2019-07-31T11:42:06.032Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=1026431000&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2019-07-31T11:42:09.774Z] Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-07-31T11:51:17.037Z] ----------------------------------------------------------------------
[debug] [2019-07-31T11:51:17.039Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\14pra\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login
[debug] [2019-07-31T11:51:17.040Z] CLI Version:   7.2.2
[debug] [2019-07-31T11:51:17.040Z] Platform:      win32
[debug] [2019-07-31T11:51:17.040Z] Node Version:  v10.16.0
[debug] [2019-07-31T11:51:17.041Z] Time:          Wed Jul 31 2019 17:21:17 GMT+0530 (India Standard Time)
[debug] [2019-07-31T11:51:17.042Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=19886601&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2019-07-31T11:51:19.665Z] Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[error] 
[error] Error: An unexpected error has occurred.
